plugins/drivers/testutils/testing.go

```diff
@@ -247,30 +248,3 @@ func (d *MockDriver) ExecTask(taskID string, cmd []string, timeout time.Duration
 func (d *MockDriver) ExecTaskStreaming(ctx context.Context, taskID string, execOpts *drivers.ExecOptions) (*drivers.ExitResult, error) {
 	return d.ExecTaskStreamingF(ctx, taskID, execOpts)
 }
-
```

Comment:
this was redundant with https://github.com/hashicorp/nomad/pull/9668/files#diff-b253f8bc653f00ffbccadd615558fe37527c07bbb9a24db14c82e90047f3a39dR78

removed this one

Reply:
update: as suspected, this was duplicated to avoid import cycle. putting it  back.